bin/open-ftl-pr.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -exo pipefail
+
+DOCKER_IMAGE=mozorg/bedrock:latest
```

Comment:
nit: Use `CI_COMMIT_SHORT_SHA` to ensure that you use the image that you just built?

Reply:
I thought about that, but to keep things fast I just went with this. It's not guaranteed that this will run on the same runner that built this image. Also it's not really important that this be run by the same exact version of the site. Maybe it would help with some issues though? I'm not sure. Hmmm...

Reply:
Yeah. But even if we rerun this was run only inside the docker container which was deleted and will start fresh again.

Reply:
If I did do that then I wouldn't have to pass in `CI_COMMIT_SHORT_SHA` in the `docker run` command either since the image would have it built in via the `GIT_SHA` var.

Reply:
Okay @glogiotatidis. Great call. I've updated things. I sourced the `set_git_env_vars.sh` script and used the `$DEPLOYMENT_DOCKER_IMAGE` variable that will be the image name of the newly pushed image which includes the branch name.